### PR TITLE
Added test for human_pose_estimation_3d_demo

### PR DESCRIPTION
**Pull request by eizamaliev**

### CODE REVIEW

**Comment by IRDonch** [comment]
Looks good, but you need to alter the test script to set `PYTHONPATH` before running the demo.

**Comment by eizamaliev** [comment]
@Wovchena please review

File: demos/tests/run_tests.py [hunk 1]
```diff
@@ -90,6 +91,8 @@ def main():
 
     num_failures = 0
 
+    os.putenv('PYTHONPATH',  "{}:{}/lib".format(os.environ['PYTHONPATH'], args.demo_build_dir))
```
Comment (Wovchena) [inline demos/tests/run_tests.py]: I suggest https://docs.python.org/3/library/sys.html#sys.path

Comment (Wovchena) [reply demos/tests/run_tests.py]: It won't work.

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
issue with clSetUserEventStatus not related to these changes

**Pull request merged**